### ISSUE TITLE
Borked CORS headers?

### ISSUE THREAD

**Issue by Souliboi** [derhuerst/db-rest#55]
When trying to use this API, or play around on the [API Playground](https://petstore.swagger.io/?url=https%3A%2F%2Fv6.db.transport.rest%2F.well-known%2Fservice-desc%0A), it will throw CORS related errors.

In the Swagger UI Playground for example it states
```
Errors

 
Fetch error
NetworkError when attempting to fetch resource. https://v6.db.transport.rest/.well-known/service-desc
Fetch error
Possible cross-origin (CORS) issue? The URL origin (https://v6.db.transport.rest) does not match the page (https://petstore.swagger.io). Check the server returns the correct 'Access-Control-Allow-*' headers.
```
In my own project I get the following error
```
Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at https://v6.db.transport.rest/stops/8010338/departures?duration=600&results=10&linesOfStops=false&remarks=true. (Reason: CORS header ‘Access-Control-Allow-Origin’ missing). Status code: 503.
```

**Comment by Souliboi** [comment]
Appeared to be a API issue from another end. Closing.

**Comment by derhuerst** [comment]
Please re-open in case you identify what this was about.

**Comment by traines-source** [comment]
Chrome does complain that the headers are invalid (also for vbb-rest, bvg-rest (and others?)). Have fixed it for db-rest now in #61 (more specifically, [ac7d86f](https://github.com/derhuerst/db-rest/pull/61/commits/ac7d86f962ed24f5b30013c436c3bad538cb9e81)).
